shared/clients/skipgo/client.go

```diff
@@ -444,7 +445,14 @@ func (s *skipGoClient) TrackTx(
 	defer resp.Body.Close()
 
 	if resp.StatusCode != http.StatusOK {
-		return "", fmt.Errorf("status code %d returned from Skip Go when submitting transaction to /track %s: %w", resp.StatusCode, txHash, handleError(resp.Body))
+		e := handleError(resp.Body)
+		if strings.Contains(e.Error(), "failed to fetch tx from chain") {
```

Comment:
not sure if this is the right way to determine a tx not found on chain @dhfang

Reply:
@nadim-az sorry for not remembering this earlier, but the track endpoint is not correctly identifying cases where the tx is not found and returning a 404, which is how we should be identifying missing transactions. let's save this work somewhere so we can reference it once it track is fixed, but  let's go back to what you had with the check at the start of the update transfers function. there are technically some edge cases around stopping and starting the solver and incorrectly abandoning a transfer, but these should have minimal impact on the function of the solver